scripts/testnet/hardfork.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+export ELRONDTESTNETSCRIPTSDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source "$ELRONDTESTNETSCRIPTSDIR/variables.sh"
+source "$ELRONDTESTNETSCRIPTSDIR/include/config.sh"
+
+# Load local overrides, .gitignored
+LOCAL_OVERRIDES="$ELRONDTESTNETSCRIPTSDIR/local.sh"
+if [ -f "$LOCAL_OVERRIDES" ]; then
+  source "$ELRONDTESTNETSCRIPTSDIR/local.sh"
+fi
```

Comment:
`local.sh` is already loaded by `variables.sh`, is it needed here?

Reply:
removed